corehq/apps/reports/static/reports/js/case_list.js

```diff
@@ -13,7 +13,7 @@ hqDefine("reports/js/case_list", ['jquery', 'analytix/js/kissmetrix', 'hqwebapp/
                                 if (item.substring(0,3) === "t__") { return userTypes[item.substring(3)]; }
                                 else { return item; }
                             }
-                        )}
+                        ).join()}
```

Comment:
Should this use `.join(",")`

Reply:
That didn't seem to change the output - I get this with or without the `","`
`filters: "project_data,admin,Web Users,u__bb3d4b70f9624df1afcd2e2b5531b91c"`

Reply:
I'd also thought this was weird, so I looked it up, and yep, comma is the default separator: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join

Reply:
maybe `","` is the default if no separator is given? interesting.

Reply:
maybe `","` is the default if no separator is given? interesting. Still learning :)